build.gradle

```diff
@@ -1,6 +1,6 @@
 plugins {
     id 'java'
-    id 'org.jetbrains.kotlin.jvm' version '1.8.20'
+    id 'org.jetbrains.kotlin.jvm' version '2.0.0'
```

Comment:
The compiler API for kotlin 2 has changed. More info below

Reply:
Why 2.0 and not 2.1 that was just released?